drivers/display/Kconfig.rm68200

```diff
@@ -3,6 +3,6 @@
 
 config RM68200
 	bool "RM68200 display driver"
-	depends on DISPLAY_MCUX_MIPI_DSI
+	depends on MIPI_DSI
```

Comment:
There should be just one commit that introduces rm68200 panel driver instead of two.

Reply:
Apologies for overloading this PR. This will get fixed once we have a MIPI API PR finalized and merged.